public/lesswrong-editor/src/ckeditor.js

```diff
@@ -156,7 +156,7 @@ const embedConfig = {
 	extraProviders: [
 		{
 			name: 'Elicit',
-			url: /^elicit.org\/binary\/questions\/(\w+)/,
+			url: /^elicit.org\/binary\/questions\/([a-zA-Z0-9-]+)/,
```

Comment:
doublechecking the final dash after the 9 is supposed to be there? I don't know regex but it surprised me.

Reply:
Yep, it means "characters from a-Z, 0-0 and dashes"

Reply:
Elicit IDs can have underscores, eg https://elicit.org/binary/questions/sI-vTZ_DL .

(Sorry for missing that they can have dashes, when I first reviewed this; I did check the regex, but looked at a sampling of Elicit IDs which happened to have underscores in it but no dashes, so I assumed it was '[a-zA-Z0-9_]' which is a subset of `\w` and is lot more common than '[a-zA-Z0-9_-]'.)